templates/ingress.yaml

```diff
@@ -5,6 +5,12 @@ metadata:
   name: {{ include "rudderstack.fullname" . }}
   labels:
     {{- include "rudderstack.labels" . | nindent 4 }}
+  {{- with .Values.backend.labels }}
```

Comment:
What is the difference between backend labels and backend.ingress labels.. Why do we need backend labels on ingress??

Reply:
Sometimes we might need labels specific to ingress or services so they get picked/watched by ingress controllers or service monitors.